packages/vkui/src/components/Footer/Footer.tsx

```diff
@@ -4,14 +4,27 @@ import { HasComponent } from '../../types';
 import { Footnote } from '../Typography/Footnote/Footnote';
 import styles from './Footer.module.css';
 
-export type FooterProps = React.AllHTMLAttributes<HTMLElement> & HasComponent;
+export type FooterProps = React.AllHTMLAttributes<HTMLElement> &
+  HasComponent & {
+    mode?: 'group' | 'list' | 'loader';
```

Comment:
Может, заодно с type на interface переделаем, раз появилось свое свойство?

```suggestion
export interface FooterProps extends React.AllHTMLAttributes<HTMLElement>, HasComponent {
  mode?: 'group' | 'list' | 'loader';
}
```

Reply:
Зря быканула (2), резолвлю в пользу отдельного issue: 
- https://github.com/VKCOM/VKUI/issues/4692